.github/workflows/wheels.yml

```diff
@@ -51,9 +51,14 @@ jobs:
         # https://github.com/github/feedback/discussions/7835#discussioncomment-1769026
         buildplat:
         - [ubuntu-20.04, manylinux_x86_64]
-        - [macos-11, macosx_*]
+        # TODO: Consider re-enabling macos wheels, once meson-python makes it easy to
+        # cross compile
+        #- [macos-11, macosx_*]
+        - [macos-11, macosx_x86_64]
         - [windows-2019, win_amd64]
-        - [windows-2019, win32]
+        # Turn off for now
+        # TODO: Re-enable after mesonbuild/meson-python#167 goes in

```

Comment:
Looks like this was merged in back in November - is this comment still accurate?

Reply:
I don't think I want to put in the effort to do the cross-compiling for 32 bit wheels anymore. Dropping wheels for all 32-bit users(we only currently provide them for Windows) is probably fine since 

I'd like to drop 32-bit support entirely sometime in the near future, but this requires discussion with numpy and other projects in the ecosystem.

Reply:
I think a lot of the WASM builds that are being developed in Python are 32 bit. If that's correct I don't know that we should fully move away, but yea this isn't a blocker for now